specification/components/stu3/schemas/endpoints/a010-patient-service-search.yaml

```diff
@@ -21,6 +21,9 @@ description: |
 
   So that I can offer the patient appropriate services that meet their referral needs
 
+  ## Important Notes
+  If using this endpoint for modifying an existing Advice Request or Referral (such as via the [[HYPERLINK_A031]] endpoint for example), you will need to use the Commissioning Rule Organisation parameter. This will ensure that the services returned in the response are valid for use with the Advice Request or Referral.
```

Comment:
As discussed confirm with BA if there is an Advice flow that needs the commissioning rule org (i.e. reshortlisting). I don't think so. You mentioned it doesn't look to be on CommunicationRequest so would need adding if there is a flow.

It may be nice to link to the A005 endpoint similar to how the important note in A0015 does.

Reply:
Altered this wording as suggested.